src/test/java/de/tum/in/www1/artemis/exercise/modelingexercise/ModelingSubmissionIntegrationTest.java

```diff
@@ -267,6 +267,7 @@ void saveAndSubmitModelingSubmission_useCaseDiagram() throws Exception {
     @Test
     @WithMockUser(username = TEST_PREFIX + "student1")
     void saveAndSubmitModelingSubmission_isTeamMode() throws Exception {
+        // todo
```

Comment:
Todo?

Reply:
I decided to do this in another PR and forgot to remove the todo :sweat_smile: